apps/prairielearn/src/pages/instructorAssessmentInstance/instructorAssessmentInstance.js

```diff
@@ -40,25 +42,35 @@ router.get(
       throw error.make(403, 'Access denied (must be a student data viewer)');
     }
     res.locals.logCsvFilename = logCsvFilename(res.locals);
-    res.locals.assessment_instance_stats = (
-      await sqldb.queryAsync(sql.assessment_instance_stats, {
+    res.locals.assessment_instance_stats = await sqldb.queryRows(
+      sql.assessment_instance_stats,
+      {
         assessment_instance_id: res.locals.assessment_instance.id,
-      })
-    ).rows;
+      },
+      assessementInstanceStatsSchema,
```

Comment:
Schema names should start with a capital letter.

Reply:
Updated schema name to `AssessmentInstanceStatsSchema` in commit fce8031.